apps/site/src/views/Home/sections/Mentor/Mentor.module.scss

```diff
@@ -0,0 +1,102 @@
+@use "bootstrap-utils" as bootstrap;
+
+.applySticky {
+    background-image: url("~@/assets/images/MentorStickyYellow.svg");
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 3rem;
+    padding-left: 5rem;
+    padding-right: 5rem;
+    padding-bottom: 8rem;
+    margin-left: 22%;
+    position: relative;
+    height: 325px;
+
+    h2 {
+        font-size: 1.8rem;
+    }
+
+    a {
+        position: absolute;
+        bottom: 10%;
+        left: 5%;
+        transform: rotate(7.5deg);
+    }
+    
+
+    @include bootstrap.media-breakpoint-down(xxl) {
+        & {
+            margin-left: 17.5%;
+        }
+    }
+}
+
+.descSticky {
+    background-image: url("~@/assets/images/MentorStickyGreen.svg");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: 60%;
+    padding: 6rem;
+    position: absolute;
+    z-index: -1;
+    top: 15%;
+    left: 40%;
+    width: 41.667%;
```

Comment:
If we're going to absolutely position but proportionally size the green sticky, the parent container will need some more bottom padding to avoid collision on certain views.